*: test failures with client TLS + Go 1.7

```
TestCtlV3PutClientTLS

=== RUN   TestCtlV3DelClientTLS
--- FAIL: TestCtlV3DelClientTLS (0.22s)
	ctl_v3_kv_test.go:200: delTest #0-0: ctlV3Put error (read /dev/ptmx: input/output error (expected "OK", got ["Error:  rpc error: code = 13 desc = transport is closing\r\n"]))
```

--- a/pkg/transport/listener.go
+++ b/pkg/transport/listener.go
@@ -208,6 +208,9 @@ func (info TLSInfo) ServerConfig() (*tls.Config, error) {
 		cfg.ClientCAs = cp
 	}
 
+	// "h2" NextProtos is necessary for enabling HTTP2 for go's HTTP server
+	cfg.NextProtos = []string{"h2"}
+
 	return cfg, nil
 }
 

